core/common/src/main/java/alluxio/client/hive/HiveCacheContext.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.hive;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+/**
+ * Data structure that stores and returns Hive related Cache Scope
+ */
+public class HiveCacheContext
+{
+  private final String mDatabase;
+  private final String mTable;
+  private final String mPartition;
+
+  public HiveCacheContext(String database, String table, String partition)
```

Comment:
The partition might be null I think in the hive context.  So put an annotation @Nullable?

Reply:
sg